### PR TITLE
rpo shim support and test parity

### PR DESCRIPTION
**Pull request by pradipm90**
rpo shim support and test parity

### CODE REVIEW

**Comment by NickGoog** [comment]
LGTM, will wait for Dilip b/c he has more familiarity with the feature

File: gslib/commands/rpo.py [hunk 1]
```diff
@@ -116,6 +121,48 @@ class RpoCommand(Command):
...
+              GcloudStorageMap(
+                  gcloud_command={
+                      'enforced':
+                          GcloudStorageMap(
+                              gcloud_command=[
+                                  'alpha',
+                                  'storage',
+                                  'buckets',
+                                  'update',
+                                  '--rpo',
```
Comment (dilipped) [inline gslib/commands/rpo.py:144]: I will have to take a look at the gcloud CL for rpo. I don't think we should have boolean flags for rpo as the set command accepts strings like ASYNC_TURBO and DEFAULT. Please ping me offline the CL for rpo.

File: gslib/commands/rpo.py [hunk 2]
```diff
@@ -116,6 +121,48 @@ class RpoCommand(Command):
...
+                  gcloud_command=[
+                      'alpha', 'storage', 'buckets', 'list', _GCLOUD_LIST_FORMAT
+                  ],
+                  flag_map={},
+                  supports_output_translation=True,
+              ),
+          'set':
+              GcloudStorageMap(
+                  gcloud_command={
+                      'enforced':
```
Comment (dilipped) [inline gslib/commands/rpo.py:137]: Did you try running the command with use_gcloud_storage=True in boto config? I think it should error out.

Comment (dilipped) [reply gslib/commands/rpo.py:137]: Things like these would also be caught by adding a shim translation test like this: https://github.com/GoogleCloudPlatform/gsutil/blob/8cbf3362f34becf358f2fcd60dc3f20bf08a9a17/gslib/tests/test_ls.py#L194

**Review by dilipped — CHANGES_REQUESTED** [review]
I think the implementation is incorrect. Feel free to reach out if you have any questions.

**Comment by dilipped** [comment]
This is being implemented in a separate PR. Closing this one.

**Pull request closed without merging**